tool/gravity/cli/status.go

```diff
@@ -378,6 +384,18 @@ func printNodeStatus(node statusapi.ClusterServer, w io.Writer) {
 	}
 }
 
+func printPrometheusAlerts(alerts []*models.GettableAlert, w io.Writer) {
+	for _, alert := range alerts {
+		if *alert.Status.State != "active" {
+			continue
+		}
+
+		duration := time.Now().Sub(time.Time(*alert.StartsAt)).Round(time.Second)
+		fmt.Fprintf(w, "    * %v [%v]\n", alert.Labels["alertname"], duration)
```

Comment:
I'm wonder if this is not going to cause indentation issues in the long run when the output would need to be readjusted for some reason. Have you considered accepting indent as argument and using tabs?

Reply:
Unless I'm missing something, I attempted to write this consistent with the rest of the status formatting:
https://github.com/gravitational/gravity/blob/master/tool/gravity/cli/status.go#L336-L360
https://github.com/gravitational/gravity/blob/master/tool/gravity/cli/status.go#L362-L379
https://github.com/gravitational/gravity/blob/master/tool/gravity/cli/status.go#L313-L334